docs/madr/001-metrics.md

```diff
@@ -0,0 +1,47 @@
+# Metrics
+
+* Status: accepted
+
+Technical Story: https://github.com/Kong/mesh-perf/issues/5
+
+## Context and Problem Statement
+
+We need to define set of metrics that we want to measure to have a consistent perf tests.
+
+## Considered Options
+
+* Set of metrics
+
+## Decision Outcome
+
+Chosen option: "Set of metrics"
+
+## Pros and Cons of the Options
+
+### Set of metrics
+
+P50, P90, P99 of
+* Time between applying Deployment and Pod be ready (exposed by Kubernetes?)
+* XDS config delivery (time between setting the config into snapshot up to receiving ACK/NACK)
+* XDS watchdog sync (time spent on generating XDS config)
+* Kubernetes reconciles (Pod to Dataplane conversion, VIP generation)
+* Latency of Kube API server responses (exposed by Kubernetes)
+* Latency of DB operations (exposed by Kuma)
+* Kuma API server responses
+* All Kuma components that have tickers (like insights, sub finalizer etc.)
+
+Number of
+* XDS reconciliations
+* Kubernetes reconciliations
+* Number of DB queries
+
+### Scenarios
```

Comment:
do we want to test how big clusters can be deployed without `reachable services`?

Reply:
I don't think it make sense. We already know that it's not scalable solution